packages/client/src/game/entities/Terrain.ts

```diff
@@ -1,9 +1,10 @@
 import { spriteCoordinates } from '../data/constants';
-import type { EntitySettings } from '../typings';
-import { EntityEvent } from './../typings/index';
+import { EntityEvent, EntitySettings, TerrainVariant } from '../typings';
```

Comment:
Можно еще такой вариант, раз ранее мы импортировали как type. Хотя в данном случае смысла в этом мало и нужно по всему проекут вообще походиться и проставлять где импортировать как type, а где нет.

```suggestion
import { type EntitySettings, type TerrainVariant, EntityEvent } from '../typings';
```

Reply:
Надо будет взять эту штуку на заметку и потом пройтись отдельно по всему проекту.